### PR TITLE
WEB-8 setup material and storybook

### PR DESCRIPTION
**Pull request by EduardValentin**

### CODE REVIEW

File: packages/lib/src/vite-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (cdanielraducu) [inline packages/lib/src/vite-env.d.ts:1]: do we need this?

File: packages/ui/src/components/theme/types.d.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+		800?: string;
+		900?: string;
+		tint50?: string;
+		tint100?: string;
+		tint200?: string;
+	}
+
+	interface ThemeOptions {}
+
+	interface TypographyVariants {
```
Comment (cdanielraducu) [inline packages/ui/src/components/theme/types.d.ts:59]: `interface TypographyVariants extends Typography {}`
Maybe you can extract the Base interface into Typography interface and then extend these ones `TypographyVariants`, `TypographyVariantsOptions` and `TypographyPropsVariantOverrides`. I think they are all the same.

File: packages/ui/src/vite-env.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (cdanielraducu) [inline packages/ui/src/vite-env.d.ts:1]: do we need this?

**Review by cdanielraducu — CHANGES_REQUESTED** [review]

File: packages/ui/src/vite-env.d.ts (hunk 3 above)
Comment (EduardValentin) [reply packages/ui/src/vite-env.d.ts:1]: they are automatically generated by vite, i understand that they are needed for IntelliSense to work with custom env variables https://vitejs.dev/guide/env-and-mode.html#intellisense-for-typescript

**Review by cdanielraducu — APPROVED** [review]

**Pull request merged**